sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -392,11 +393,18 @@ def list_accelerators(
     new_infos = defaultdict(list)
     for info in acc_infos:
         assert pd.isna(info.instance_type) and pd.isna(info.memory), acc_infos
+        if info.accelerator_name.startswith('tpu'):
+            new_infos[info.accelerator_name].append(
+                info._replace(instance_type='TPU-VM',))
+            continue
```

Comment:
Will this set the instance type of an TPU Node to TPU-VM too?

Reply:
I am not sure how the nomenclature works really. But looking at the https://github.com/skypilot-org/skypilot/pull/1758 it seems like as long as `'tpu'` is a part of the accelerator name string, it would set `use_tpu` to be True in resources.py which checks in this case if the instance_type is equal to '`TPU-VM'`

Reply:
Fixed!